pdr_backend/pdr_dashboard/dash_components/plots.py

```diff
@@ -28,6 +29,14 @@ def process_payouts(payouts: List[dict]) -> tuple:
         profit += profit_change
         correct_predictions += p["payout"] > 0
 
+        acc_l, acc_u = proportion_confint(count=correct_predictions, nobs=predictions)
+
+        acc_intervals.append(
+            {
+                "acc_l": acc_l,
+                "acc_u": acc_u,
+            }
+        )
```

Comment:
can we do this steps only if `len(predictoors) == 1 and len(feeds) == 1` ?

Reply:
I am going to add a flag for this calculation to the `process_payouts` function